Server/routes/authRoute.js

```diff
@@ -23,6 +26,7 @@ router.post(
   userEditController
 );
 router.get("/users/admin", checkAdminController);
+router.delete("/user/:userId", verifyJWT, verifyOwnership(User, "userId") , deleteUserController);
```

Comment:
We can remove the `:userId` param from this route, it is no longer required

`router.delete("/user", verifyJWT, verifyOwnership(User, "userId") , deleteUserController);`

is sufficient

Reply:
Fixed

Reply:
Let's merge this thing and then figure out what to do next

Reply:
I'm facing a couple of errors now. please text login, register, and delete to check them out

Reply:
Can you elaborate more on the errors?

How do we replicate them?

Reply:
Don't need to do anything!
Just do make a simple request (Register, Login, and Delete) (On both UI and postman) and errors will be logged on console
The error is for parts that I haven't worked on before, so I'm afraid modifying the code blindly may cause issues.